Makefile

```diff
@@ -63,3 +69,7 @@ push-images-all: push-images-version push-images-latest
 lint:
 	# installation: https://golangci-lint.run/usage/install/#local-installation
 	golangci-lint run --new-from-rev=${UPSTREAM_BRANCH}
+
+.PHONY: mock
+mock:
+	go generate mock_gen.go
```

Comment:
maybe add one more line and delete mock_gen.go ?
```
go get github.com/golang/mock/mockgen
mockgen -source=plugin/output/s3/s3.go -destination=plugin/output/s3/mock/s3.go
```

Reply:
I tried both this method and declaring mockgen in Makefile, both are fine.
Seems fine, is it really an issue?

Reply:
Done